conf/nginx.conf

```diff
@@ -2,10 +2,10 @@
 # Either include it in the default mime.types list
 # or include you can include that list explicitly and add the file extension
 # only for Nextcloud like below:
-include mime.types;
-types {
-    text/javascript js mjs;
-}
+#include mime.types;
+#types {
+#    text/javascript js mjs;
+#}
```

Comment:
hi!
why have you commented this?

Reply:
These lines was not here on v27.1.4, they have been added after a bug with the v28...
https://github.com/nextcloud/server/issues/42989

Reply:
oh, i see, thanks!

Reply:
And also there is a bug https://github.com/nextcloud/server/issues/42989

Reply:
I have to be careful with links...

Reply:
I think this line is enough to accept `mjs` files
https://github.com/YunoHost-Apps/nextcloud_ynh/blob/96e835d9081730f09b2545680943c5790fce114f/conf/nginx.conf#L140

Reply:
I was also perhaps because of this discussion just before merge to 28
https://github.com/YunoHost-Apps/nextcloud_ynh/pull/622

Reply:
You are perhaps rigth, seems is better keep it reading the link to the official documentation https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html#nextcloud-in-a-subdir-of-the-nginx-webroot

Reply:
I keep it, ok